examples/7_get_started_with_real_robot.md

```diff
@@ -0,0 +1,998 @@
+    <p style="text-align:center;">3. Rest position</p>
+  </div>
+</div>
+
+And here are the corresponding positions for the leader arm:
+
+<div style="display:flex; justify-content:center;">
+  <div style="width:30%; margin:10px;">
+    <img src="../media/koch/leader_zero.webp?raw=true" alt="Koch v1.1 leader arm zero position" title="Koch v1.1 leader arm zero position" style="width:100%;">
+    <p style="text-align:center;">1. Zero position</p>
+  </div>
+  <div style="width:30%; margin:10px;">
+    <img src="../media/koch/leader_rotated.webp?raw=true" alt="Koch v1.1 leader arm rotated position" title="Koch v1.1 leader arm rotated position" style="width:100%;">
+    <p style="text-align:center;">2. Rotated position</p>
+  </div>
+  <div style="width:30%; margin:10px;">
+    <img src="../media/koch/leader_rest.webp?raw=true" alt="Koch v1.1 leader arm rest position" title="Koch v1.1 leader arm rest position" style="width:100%;">
+    <p style="text-align:center;">3. Rest position</p>
+  </div>
+</div>
+
+You can watch a [video tutorial of the calibration procedure](https://youtu.be/8drnU9uRY24) for more details.
+
+During calibration, we count the number of full 360-degree rotations your motors have made since they were first used. That's why we ask yo to move to this arbitrary "zero" position. We don't actually "set" the zero position, so you don't need to be accurate. After calculating these "offsets" to shift the motor values around 0, we need to assess the rotation direction of each motor, which might differ. That's why we ask you to rotate all motors to roughly 90 degrees, to mesure if the values changed negatively or positively.
+
+Finally, the rest position ensures that the follower and leader arms are roughly aligned after calibration, preventing sudden movements that could damage the motors when starting teleoperation.
+
+Importantly, once calibrated, all Koch robots will move to the same positions (e.g. zero and rotated position) when commanded.
+
+Run the following code to calibrate and connect your robot:
+```python
+robot.connect()
+```
+
+The output will look like this:
+```
+Connecting main follower arm
+Connecting main leader arm
+Missing calibration file '.cache/calibration/koch.pkl'. Starting calibration procedure.
+
+Running calibration of main follower...
+
+Move arm to zero position
+[...]
+Move arm to rotated position
+[...]
+Move arm to rest position
+[...]
+
+Running calibration of main leader...
+
+Move arm to zero position
+[...]
+Move arm to rotated position
+[...]
+Move arm to rest position
+[...]
+
+Calibration is done! Saving calibration file '.cache/calibration/koch.pkl'
+```
+
+*Verifying Calibration*
+
+Once calibration is complete, you can check the positions of the leader and follower arms to ensure they match. If the calibration was successful, the positions should be very similar.
```

Comment:
I think we need to be careful with advice. Because after calibration I reran the script and the follower arm was put into torque mode meaning I can't move it manually. I was careful enough to notice this.

I think `robot.connect()` should be more explicit about the fact that it does this. Maybe there should be a kwarg that you have to provide to set the follower arm to torque mode. Or it should print a warning when the arm goes into torque mode.

Reply:
I think a print would be enough, no?

```python
# Enable torque on all motors of the follower arms
for name in self.follower_arms:
    print(f"Activating torque on {name} follower arm.")
    self.follower_arms[name].write("Torque_Enable", 1)
```

Reply:
yep